karma.conf.js

```diff
@@ -50,6 +50,17 @@ const SAUCE_LAUNCHERS = {
 
 const customLaunchers = Object.assign({}, USING_SL ? SAUCE_LAUNCHERS : {}, HEADLESS_LAUNCHERS);
 const browsers = USING_TRAVISCI ? Object.keys(customLaunchers) : ['Chrome'];
+const istanbulInstrumentLoader = {
+  use: [{
+    loader: 'istanbul-instrumenter-loader',
+    options: {esModules: true},
+  }],
+  exclude: [
+    /node_modules/,
+    /adapter.js/,
```

Comment:
Would these need to be `[jt]s` to continue ignoring these as they're converted?

Also, would a `$` at the end for the adapter/constants regexps be a good idea?